tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Session.java

```diff
@@ -490,6 +490,19 @@ public void run(Op op) {
     runner().addTarget(op.op()).run();
   }
 
+
+  /**
+   * Execute the graph's initializers.
+   *
+   * <p>This method is equivalent to {@code session.runner().addTarget(Ops.create(session.graph()).init()).run()}.
```

Comment:
In fact, it is more a replacement for `session.run(Ops.create(session.graph).init())`